packages/pixeloven-webpack/types/package.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "@pixeloven-webpack/types",
```

Comment:
I created this to try and help with whatever type issues I was having but it didn't fix the issue. However I did ultimately like how it centralized the types for this group of packages. Going to try this for a while and if it works out might also adopt something similar for the other workspaces.